src/core/tests/visitors/op/squeeze.cpp

```diff
@@ -6,7 +6,16 @@
 
 #include "unary_ops.hpp"
 
+namespace v0 {
 using Types = ::testing::Types<UnaryOperatorType<ov::op::v0::Squeeze, ov::element::f32>,
                                UnaryOperatorType<ov::op::v0::Squeeze, ov::element::f16>>;
 
 INSTANTIATE_TYPED_TEST_SUITE_P(visitor_without_attribute, UnaryOperatorVisitor, Types, UnaryOperatorTypeName);
+}  // namespace v0
+
+namespace v15 {
+using Types = ::testing::Types<UnaryOperatorTypeWithAttribute<ov::op::v15::Squeeze, ov::element::f32>,
+                               UnaryOperatorTypeWithAttribute<ov::op::v15::Squeeze, ov::element::f16>>;
+
+INSTANTIATE_TYPED_TEST_SUITE_P(visitor_single_attribute, UnaryOperatorVisitor, Types, UnaryOperatorTypeName);
```

Comment:
Squeeze may have 2 inputs, looking on that test case, only single input + attribute is covered by tests